### DISCUSSION TITLE
Email history log

### DISCUSSION THREAD

**Discussion by barchard** [duncanmcclean/simple-commerce#711] [Feature Requests]
### What would you like to see added?

An Email history of items sent to a Customer. This will help troubleshoot issues to know what communications have been sent to a customer (and potentially resend one if needed)

### Is there a workaround you can use in the meantime?

No

### Priority wise - where's this feature for you?

Nice to have

**Comment by duncanmcclean** [comment]
This has been included in the latest release for emails relating to order/payment statuses (https://github.com/duncanmcclean/simple-commerce/pull/956).
